handler/handler.go

```diff
@@ -7,18 +7,23 @@ import (
 )
 
 // CreateNATSQueue ready for asynchronous processing
-func CreateNATSQueue(address string, port int, clusterName string, clientConfig NATSConfig) (*NATSQueue, error) {
+func CreateNATSQueue(address string, port int, clusterName, channel string, clientConfig NATSConfig) (*NATSQueue, error) {
```

Comment:
Is this a breaking change for the gateway code? https://github.com/openfaas/faas/blob/365f459b3f3a05ad17ce83f4edd86f4276af354a/gateway/server.go#L138

Reply:
Yes, just like the one introduced in #71.